optics_functions/coupling.py

```diff
@@ -54,17 +57,16 @@ def coupling_via_rdts(df: TfsDataFrame, complex_columns: bool = True, **kwargs):
 
     if not complex_columns:
         df_res = split_complex_columns(df_res, COUPLING_RDTS)
-
     return df_res
 
 
-def coupling_via_cmatrix(df: TfsDataFrame, complex_columns: bool = True,
-                         output: Sequence[str] = ("rdts", "gamma", "cmatrix")):
-    """ Calculates C matrix then Coupling and Gamma from it.
+def coupling_via_cmatrix(df: DataFrame, complex_columns: bool = True,
```

Comment:
The name is a bit confusing because all quantities included in this function including the input describes the coupling. It is simply a way to go from MAD-X convention to most commonly used C-matrix and RDT. Maybe something like coupling_from_MADX_convension.